client/src/components/SearchInput/SearchInput.module.scss

```
@@ -0,0 +1,15 @@
+.SearchInputContainer{
+    position: relative;
+    width: 215px;
+    .SearchInput{
```

Add line spaces and white spaces for better formatting.
```suggestion
    
    .SearchInput {
```